### ISSUE TITLE
How to caculate the ground truth of the physical parameters in reality?

### ISSUE THREAD

**Issue by avaa999** [Boey-li/AdaptiGraph#9]
Hi! Thank you for providing such a perfect job. While reading your paper, I'm curious how you obtain the  values of the physical parameters, such as stiffness and friction?

**Comment by Boey-li** [comment]
Thank you for your interest in our work!

We describe our approach for obtaining the values of the physical parameters in **Section III-C (Few-Shot Physical Property Adaptation)** of the paper. In short, we utilize few-shot curiosity-driven interaction combined with gradient-free optimization techniques like Bayesian Optimization or CMA-ES to estimate the physical properties.

The code is also available—please refer to the "Inference and Planning" section in the README.md for more details.

**Comment by avaa999** [comment]
Thank you very much for your assistance, and I apologize for my delayed response. From your explanation, is the stiffness of each material derived from parameter estimation (parameter optimization)? In other words, do the label value of the stiffness serve to minimize the prediction loss of the dynamic model?

**Comment by avaa999** [comment]
Hi, Boey! I would like to clarify my question further. In the "Inference and Planning" section of the README.md, it states, "The result should show that the estimated parameter is around 0.04, indicating low granularity. This aligns with the ground truth since the object is a pile of coffee beans, which have small granular sizes." What confuses me is how to obtain or calculate the ground truth. Thanks a lot again!

**Comment by Boey-li** [comment]
Apologies for the delayed response. In the training data, the physical property values are normalized to a range between 0 and 1. For instance, smaller values correspond to lower granularity or stiffness. To achieve adaptation, we optimize the physical property values to align with the variations in the simulation.

There is no real "ground truth" for such value in the real-world, or let's say that the "ground truth" is human's impression. For example, for coffee beans, the granularity is relatively small so it should be close to the 0 value. Thus, we consider 0.04 is a reasonable value of granularity for coffee bean. You can also check out the figure 6 in our paper.

**Comment by avaa999** [comment]
Thank you for the clarification. It makes sense that the "ground truth" in this context is somewhat subjective, based on human judgment. I’ll also take a look at figure 6 in your paper for more insight. Thanks again for your help!
